docs/code/readme_standards.md

```diff
@@ -3,100 +3,135 @@
 MDC Web component documentation serves two purposes:
 
 * Populate the [Catalog Site](https://material.io/components/web/catalog/)
-* Highlight how to use our APIs, for our external developers (see Usage section)
-
-> Developers should not have to read our code to know how to use our APIs
+* Highlight how to use our APIs, for our external developers.
 
 ## README Template
 
 Please follow the [README template](readme_template.md) *exactly*. This format is
 read/understood by the [Catalog Site](https://material.io/components/web/catalog/)
 
-## Usage
-
-Under the 'Usage' section, you will document all
+## Basic Usage
 
-* HTML Structure
-* CSS Classes
-* JavaScript
-* Sass Variables and Mixins
+The 'Basic Usage' section documents the bare minimum to get the most basic version of this component up and running, including:
+* HTML structure
+* Style import
+* JavaScript import and instantiation, if applicable
 
+~~~
 ### HTML Structure
 
-HTML Structure is documented with a code snippet, e.g.
-
 ```html
 <button class="mdc-button">Foo</button>
 ```
+~~~
 
-Any additional optional HTML should be documented in separate code snippets.
-
-### CSS Classes
+In the 'Styles' subsection, include any mixin invocations necessary to make the component look as desired.
+~~~
+### Styles
 
-CSS Classes are documented in a [tabular format](#tabular-format)
+```scss
+@import "@material/button/mdc-button";
 
+// Include necessary mixin invocations here
 ```
-CSS Class | Description
---- | ---
-`mdc-foo` | Sets the foo
+~~~
+
+In the 'JavaScript Instantiation' subsection, include a note about importing a JavaScript component with a link to our importing documentation.
+~~~
+### JavaScript Instantiation
+
+```js
+import {MDCButton} from '@material/button';
+
+const button = new MDCButton(document.querySelector('.mdc-button'));
 ```
 
-### Sass Variables and Mixins
+>See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+~~~
 
-Sass variables are documented in a tabular format (See Tabular Format section).
+## Variants
 
+If the component has variants, add a subsection for each variant, under the 'Variants' section. Include a short description of the variant along with the necessary HTML structure.
+
+~~~
+### Foo Button
+
+Short description of Foo Button.
+
+```html
+<button class="mdc-button mdc-button--foo">Foo</button>
 ```
-Variable | Description
+~~~
+
+## Style Customization
+
+CSS Classes are documented in a [tabular format](#tabular-format).
```

Comment:
We've got a mix of "in a tabular format" and "in tabular format" across this file. Pick one?